modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,144 @@
+
+Control plane ({gitops-shortname} control plane)::
+In the {gitops-shortname} context, you can have a control plane for every Argo CD you install. A {gitops-shortname} control plane is any namespace where you can install an Argo CD. This control plane allows you to provision, manage, and operate Argo CD across networks, instances, and clusters. 
++
+Within a control plane namespace, Argo CD maintains a set of following Kubernetes resources, which define the continuous deployment between the source Git repository and destination clusters:
+
+* Argo CD `Application` CRs
+* `ConfigMap` API objects
+* `Secret` objects representing the {gitops-shortname} repository credentials and cluster credentials for deployment targets
+
+`openshift-gitops` is the control plane namespace for the default Argo CD instance.
+
+Declarative setup::
+A declarative description of the infrastructure required in your specified environment, for system and application setup or configuration. You can specify this description in a YAML configuration file in the Git repository. The declarative setup contains an automated process to make your environment and infrastucture match the described state. Examples include defining Argo CD applications, projects, and settings declaratively by using YAML manifests.
+
+Default Argo CD instance (Default cluster-scoped instance)::
+A default instance that a {gitops-title} Operator instantiates immediately after its installation, in the `openshift-gitops` namespace, with additional permissions for managing certain cluster-scoped resources.
+
+{gitops-shortname}::
+A declarative way to implement continuous deployment for cloud native applications. In {gitops-shortname}, a Git repository contains deployment resources, which Argo CD then keeps synchronizing with its cluster state.
+
+{gitops-shortname} CLI ({gitops-shortname} `argocd` CLI)::
+A tool to configure and manage {gitops-title} and Argo CD resources from the command line. 
+
+Instance scopes::
+Modes that determine how you want to operate an Argo CD instance. The available modes are cluster-scoped instance and namespace-scoped instance.
+
+Live state::
+The live state of application resources on a target cluster.
+
+Local cluster::
+A cluster where you install Argo CD.
+
+Manifest::
+In the {gitops-shortname} context, a manifest is a YAML representation of Kubernetes resources defined within a {gitops-shortname} repository, with the intent to deploy those resources to a target Kubernetes cluster. Examples include the YAML representation of resources such as `Deployment`, `ConfigMap`, or `Secret`.
+
+Multitenancy::
+A software architecture where a single software instance serves multiple yet distinct user groups.
+
+Namespace-scoped instance (Application delivery instance)::
+A mode in which Argo CD is configured to only manage resources in certain namespaces on a cluster and use them for application delivery.
```

Comment:
What is _them_? Resources? 
Also, the _only_ should be as close to the term it modifies as possible. 
```suggestion
A mode in which an Argo CD instance is configured to manage resources in only certain namespaces on a cluster and use the resources for application delivery.
```